enclave/core/atexit.c

```diff
@@ -41,10 +41,11 @@ static oe_spinlock_t _spin = OE_SPINLOCK_INITIALIZER;
 
 static oe_atexit_entry_t* _new_atexit_entry(void (*func)(void*), void* arg)
 {
+    extern void* oe_nodebug_malloc(size_t);
```

Comment:
Rather than copying the same prototypes into multiple .c files (here, debugmalloc.c, sgx/backtrace.c, etc.), can you put them into a header file like oe_alloc_thread.h?   Want to avoid different files being out of sync if the prototype ever changes in the future.

Reply:
Sure, the signatures are those of malloc/realloc/..., so I don't expect them to change any time soon, but that's fine.

Reply:
Done